org.civicrm.bootstrapcivihr/scss/overrides/_variables.scss

```diff
@@ -14,3 +14,4 @@ $steps-height: 24px;
 $steps-compact-height: 28px;
 
 $steps-gutter: ($steps-height / 4);
+$spinner: ($steps-height / 4);
```

Comment:
what is this for, you don't seem to be using it anywhere else and it wasn't here in the previous commits

Reply:
I am not sure why its changed!  i will fix it. I wont be needing it.